File: CTDL1/New folder/Common7/IDE/ItemTemplatesCache/VisualBasic/Office/1033/VSTOFormRegionControlV4/ImportedFormRegion.vb
Public Class $safeitemrootname$

#Region "Form Region Factory"

$formregionattributes$    Partial Public Class $safeitemrootname$Factory

    Private Sub InitializeManifest()
            Dim resources As System.Resources.ResourceManager = New System.Resources.ResourceManager(GetType($safeitemrootname$))
$manifestinitialization$
    End Sub

    ' Occurs before the form region is initialized.
    ' To prevent the form region from appearing, set e.Cancel to true.
    ' Use e.OutlookItem to get a reference to the current Outlook item.
        Private Sub $safeitemrootname$Factory_FormRegionInitializing(ByVal sender As Object, ByVal e As Microsoft.Office.Tools.Outlook.FormRegionInitializingEventArgs) Handles Me.FormRegionInitializing

    End Sub

    End Class

#End Region

    ' Occurs before the form region is displayed.
    ' Use Me.OutlookItem to get a reference to the current Outlook item.
    ' Use Me.OutlookFormRegion to get a reference to the form region.
    Private Sub $safeitemrootname$_FormRegionShowing(ByVal sender As Object, ByVal e As System.EventArgs) Handles MyBase.FormRegionShowing

    End Sub

    ' Occurs when the form region is closed.
    ' Use Me.OutlookItem to get a reference to the current Outlook item.
    ' Use Me.OutlookFormRegion to get a reference to the form region.
    Private Sub $safeitemrootname$_FormRegionClosed(ByVal sender As Object, ByVal e As System.EventArgs) Handles MyBase.FormRegionClosed

    End Sub

End Class

File: CTDL1/New folder/Common7/IDE/ItemTemplatesCache/CSharp/Office/1033/VSTOFormRegionControlV4/ImportedFormRegion.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Resources;
using System.Text;
using Office = Microsoft.Office.Core;
using Outlook = Microsoft.Office.Interop.Outlook;

namespace $rootnamespace$
{
    public partial class $safeitemrootname$
    {
        #region Form Region Factory 

        $formregionattributes$ public partial class $safeitemrootname$Factory
        {
            private void InitializeManifest()
            {
                ResourceManager resources = new ResourceManager(typeof($safeitemrootname$));
                $manifestinitialization$
            }

            // Occurs before the form region is initialized.
            // To prevent the form region from appearing, set e.Cancel to true.
            // Use e.OutlookItem to get a reference to the current Outlook item.
            private void $safeitemrootname$Factory_FormRegionInitializing(object sender, Microsoft.Office.Tools.Outlook.FormRegionInitializingEventArgs e)
            {
            }
        }

        #endregion

        // Occurs before the form region is displayed.
        // Use this.OutlookItem to get a reference to the current Outlook item.
        // Use this.OutlookFormRegion to get a reference to the form region.
        private void $safeitemrootname$_FormRegionShowing(object sender, System.EventArgs e)
        {
        }

        // Occurs when the form region is closed.
        // Use this.OutlookItem to get a reference to the current Outlook item.
        // Use this.OutlookFormRegion to get a reference to the form region.
        private void $safeitemrootname$_FormRegionClosed(object sender, System.EventArgs e)
        {
        }
    }
}
